bindings/java/src/main/com/apple/foundationdb/FDBTransaction.java

```diff
@@ -257,6 +277,26 @@ public void setReadVersion(long version) {
 		}
 	}
 
+	@Override
+	public CompletableFuture<Long> getEstimatedRangeSizeBytes(byte[] begin, byte[] end) {
+		pointerReadLock.lock();
+		try {
+			return new FutureInt64(Transaction_getEstimatedRangeSizeBytes(getPtr(), begin, end), executor);
+		} finally {
+			pointerReadLock.unlock();
+		}
+	}
+
+	@Override
+	public CompletableFuture<Long> getEstimatedRangeSizeBytes(Range range) {
+		pointerReadLock.lock();
```

Comment:
This could optionally be written more concisely as `return getEstimatedRangeSizeBytes(range.begin, range.end)`.

Reply:
Yeah that's better.

Reply:
Done.